### DISCUSSION TITLE
use same field "Checked Out To" in assets for accessories, licenses etc

### DISCUSSION THREAD

**Discussion by ghost** [snipe/snipe-it#11522] [General]
Is there any way to use the same field Checked Out To when viewing accessories, license etc the same way that you can see the assigned user when looking at Assets? I know that you can go to each user and see what they have assigned on them but I would like to see this in accessories view also. I have tried to create a custom field for assigning users but haven't found what values to use for it.

I have searched for this but haven't found any way to see this, checked the db tables as well but no luck. It's not the end of the world if it can't be done but it would be a big plus.

Thanks beforehand

**Comment by snipe** [comment]
I'm not sure I understand what you mean? You can see the list of users in all of those items. 

<img width="1617" alt="Screen Shot 2022-07-27 at 12 04 37 PM" src="https://user-images.githubusercontent.com/197404/181352122-f417639d-b4c3-4477-920b-7e1e47a0dfb0.png">

**Comment by ghost** [comment]
Yes but I wonder if there is a way to see the user from accessories view, according to pic below. It's not a big issue but would be nice if it was possible.
![image](https://user-images.githubusercontent.com/109520711/181779581-54b98bfa-33bf-4940-82fc-02747d8a1f24.png)

**Reply by mikeroq** [reply]
Well that's the overall accessory table. Since they have a QTY it's kinda hard to show what one is checked out to. Say you had 10 quantity on one item, and all 10 were checked out to 10 different people. That would take a lot of space in the table. Imagine if you had 100 QTY checked out you would have a massive table row to display all that.

To properly see all the check outs you want to view the accessory like the screenshot snipe sent above.

**Comment by ghost** [comment]
I understand, thanks for the responses and help for this matter @snipe @mikeroq
